src/Utils.hs

```diff
@@ -63,12 +52,40 @@ subsetSizeDiff = 10.0
 epsd :: Floating a => a -- to prevent 1/0 (infinity). put it in the denominator
 epsd = 10 ** (-10)
 
+-- TODO: check naming convention of this
+labelName :: String -> String
```

Comment:
i changed the internal naming scheme to “subObjName styObjName [label]” (e.g. "A xaxis", "A xaxis label"); see Utils in my branch)